src/Map/ReactMap.tsx

```diff
@@ -16,10 +16,6 @@ const center = {
   lng: -122.315
 };
 
-// Statically define libraries to avoid the "LoadScript has been reloaded
-// unintentionally!" performance warning.
-const LIBRARIES: ("places")[] = ["places"];
```

Comment:
Is it not important to keep filtering to these libraries?

Reply:
oh dang i misread that note and assumed it was about the whole line but it was just about the latter part

Reply:
added it back in :) 51ce1d8